docs/docs/spec.md

```diff
@@ -392,7 +386,9 @@ condition is met, skip the remaining conditions.
 
 > If tool authors prefer, they can safely produce `MINUS_NULL` in any case in
 > which it is equivalent to `NO_CHANGE`. For example, there is no difference
-> between `String NO_CHANGE` and `String MINUS_NULL`.
+> between `Foo NO_CHANGE` and `Foo MINUS_NULL` for any class type `Foo` (nor for
+> any array type or the null type). The difference *is* significant for
+> intersection types, type variables, and probably union types.
```

Comment:
"Probably"? 😆

Reply:
I mean, it's not like I'd thought about union types at all until Friday :) But hey, it's a non-normative comment, so let's be bold!